### PR TITLE
feat(sort): Add enableRemoveSort option to GridOptions

### PR DESCRIPTION
**Pull request by richardm**

### CODE REVIEW

File: src/js/core/factories/GridOptions.js [hunk 1]
```diff
@@ -362,6 +362,15 @@ angular.module('ui.grid')
 
       /**
        * @ngdoc boolean
+       * @name suppressRemoveSort
+       * @propertyOf ui.grid.class:GridOptions
+       * @description False by default. When enabled, this setting limits all
+       * columns to sorting in asc or desc order and removes the "no sort" order.
+       */
+      baseOptions.enableSorting = baseOptions.enableSorting !== true;
```
Comment (swalters) [inline src/js/core/factories/GridOptions.js]: shouldn't this be baseOptions.suppressRemoveSort?

Comment (richardm) [inline src/js/core/factories/GridOptions.js]: Yeah, just fixed that and a couple other issues. Should work this time.

**Comment by swalters** [comment]
@richardm can you fix the merge conflicts and then rebase your branch to squash the commits? thanks

**Comment by benheymink** [comment]
@swalters I'm after this fix to, I'm happy to submit a new PR with @richardm 's changes if thats OK?

**Comment by mportuga** [comment]
@benheymink I would be more than happy to review a new PR with these changes. Closing this for now since it appears the developer that opened it has since abandoned it.

**Pull request closed without merging**